code/datums/cleaning.dm

```diff
@@ -0,0 +1,81 @@
+//For handling standard click-to-clean items like soap and mops.
+/datum/proc/cleaning_act(atom/target, mob/user, atom/cleaner, cleanspeed = 50, ismop = FALSE)
```

Comment:
why is this scoped to `/datum/` I dont think we will ever need to clean someones mind.

Reply:
this should be scoped to atom or obj